content/zh/docs/setup/keadm_zh.md

```diff
@@ -47,18 +51,19 @@ Kubernetes version verification passed, KubeEdge installation will start...
 ...
 KubeEdge cloudcore is running, For logs visit:  /var/log/kubeedge/cloudcore.log
 ```
+当您看到以上信息，说明 KubeEdge 的云端组件 cloudcore 已经成功运行。
 
 ### keadm beta init
-
-现在可以使用 `keadm beta init` 进行云端组件安装。
+如果您想要使用容器化方式部署云端组件 cloudcore ，您可以使用 `keadm beta init` 进行云端组件安装。
```

Comment:
after v1.11, `keadm init` will install the cloudcore with container. this sentence should be modified.